floater/generators.py

```diff
@@ -97,13 +97,34 @@ def parcel_area(self, latlon=True):
             return dx * dy
 
 
-    def to_mitgcm_format(self, filename, tstart=0, mesh='rect'):
-    	#xx, yy = np.meshgrid(x, y)
+    def to_mitgcm_format(self, filename, tstart=0, iup=0, mesh='rect'):
+	"""Output floatset in MITgcm format
+	PARAMETERS
+	----------
+	filename : The filename to save the floatset data in 
+		   (e.g.float.ini.pos.hex.bin)
+	tstart : time for float initialisation (default = 0)
+	iup : flag if the float
```

Comment:
use numpydoc syntax for discrete options:
https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt